### DISCUSSION TITLE
Customize icecast.xml

### DISCUSSION THREAD

**Discussion by gAlleb** [AzuraCast/AzuraCast#6670] [Q&A] [answered]
Hello there!

I was wandering is it possible to customize icecast.xml?

I'm talking specifically about ` <source-timeout>10</source-timeout>`

I've tried editing, but config is automatically renewed on startup. Then there is `Custom Configuration` in Stations Profile but it doesn't allow to change, only add.

Thanks!

**Comment by BusterNeece** [comment, marked as answer]
Currently you can only supplement the auto-generated config in the Custom Configuration, though it should also allow you to override existing configs in most cases.

**Comment by gAlleb** [comment]
Thanx @BusterNeece 
So to make it clear: if I supplement the auto-generated config with `<source-timeout>30</source-timeout>` would this override the default `10sec`?

**Reply by BusterNeece** [reply]
Off the top of my head, I'm not sure but I believe so yes.

**Reply by gAlleb** [reply]
Then I'll give it a try. Thanks!

**Reply by gAlleb** [reply]
@BusterNeece Indeed, It did!) Many Thanks!

**Reply by druff99** [reply]
do i have to restart the server to come into affect? i have performed and dont see any results

**Reply by gAlleb** [reply]
@druff99, sure you do.
